src/components/tabs/TabPanel/index.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
+
+var React = require('react');
+
+module.exports = React.createClass({
+  displayName: "TabPanel",
+  propTypes: {
+    content: React.PropTypes.string.isRequired,
+    index: React.PropTypes.number.isRequired,
+    active: React.PropTypes.bool.isRequired,
+    panelId: React.PropTypes.string.isRequired,
+    labelledBy: React.PropTypes.string.isRequired
+  },
+
+  render: function() {
+    var active = this.props.active ? " TabPanel--active" : '';
+
+    return (
+      <div
+        className={ "TabPanel" + active }
+        dangerouslySetInnerHTML={{ __html: this.props.content }}
```

Comment:
Instead of doing this, which prevents you from being able to use React components within the tab structure, use this.props.children, and reconfigure the interface of the top-level Tab component to accept children. Children are just an array, so you can still loop through them, create tabs for them, wrap them in other components, and so on. This would simplify your tab component to be usable like so: 
```
<Tabs>
    <Page 1/>
    <Page 2/>
    <Page 3/>
</Tabs>
```
Instead of having to convert everything to a javascript object to be passed in with the data prop.